backend/utils.py

```diff
@@ -33,3 +37,42 @@ def create_transaction(db):
             pass
 
     return db.transaction()
+
+
+class AppLogger(object):
+
+    @staticmethod
+    def log_info(args, response):
+        logger.info(
+            f"{reqparse.request.method} - "
```

Comment:
does `reqparse` contain info about the current request? I thought that was part of the `request` global variable in Flask

Reply:
Yes, `reqparse.request` contains the flask request object.

Reply:
But, for clarity, I pushed the changes to use flask request object directly instead of `reqparse`.